packages/terra-open-source-scripts/src/terra-cli/release/index.js

```diff
@@ -0,0 +1,11 @@
+const releaseMonoRepoHandler = require('./releaseMonoRepoHandler');
+const releaseRepoHandler = require('./releaseRepoHandler');
+const isMonoRepo = require('../../utils/isMonoRepo');
+
+const release = {
+  command: 'release',
+  describe: 'Tag the repo and release',
+  handler: async () => (await isMonoRepo() ? releaseMonoRepoHandler() : releaseRepoHandler()),
```

Comment:
IMO we should require the one running the command to pass a flag if it's a mono repo project instead to trying to check if a lerna.json file exists

Reply:
If you're a mono repo you already have lerna, if your not this won't get called. I'm ok with this being loosely coupled.

Reply:
I'd rather keep it simple and detect if we're in a mono repo. I use this same logic in terra dev site to determine which files to pull from.